tests/link/jax/test_random.py

```diff
@@ -443,6 +443,28 @@ def test_random_categorical():
     np.testing.assert_allclose(samples.mean(axis=0), 6 / 4, 1)
 
 
+@pytest.mark.parametrize(
+    "parameters",
+    [
+        np.array([1.0, 1.0, 1.0]),
+        np.array([1.0, 0.5, 2.0]),
+    ],
+)
+def test_random_gengamma(parameters):
+    alpha, lam, p = parameters
+    n_samples = 20_000
+    rng_state = np.random.RandomState(123)
+    rng = shared(rng_state)
+    g = at.random.gengamma(*parameters, size=(n_samples,), rng=rng)
+    g_fn = function([], g, mode=jax_mode)
+    samples = g_fn()
+    ref = stats.gengamma(alpha / p, p, scale=lam)
+    ref.random_state = rng_state
+    samples_ref = ref.rvs(n_samples)
+    test = stats.cramervonmises_2samp(samples, samples_ref)
```

Comment:
Looks like this is failing in CI because [the Python 3.7 tests are using SciPy 1.61 and not > 1.7.0](https://github.com/aesara-devs/aesara/actions/runs/4237436612/jobs/7363445156#step:5:200).

Reply:
We can probably change the SciPy lower bound to 1.7.0, because our NumPy lower bound is 1.17, and [SciPy recommends a version greater than 1.7.0 for that](https://docs.scipy.org/doc/scipy/dev/toolchain.html#numpy).  The only relevant SciPy constraint of which I'm aware is in `numba-scipy`, which currently upper-bounds SciPy at <= 1.7.3, but allows the desired 1.7.0 lower bound.

Reply:
I just tried and the tests pass locally with SciPy==1.7.2.

Reply:
The alternative is to add the test to `test_random_RandomVariable` which uses `stats.cramervonmisses`.

Reply:
I didn't use `stats.cramervonmises` in `test_random_RandomVariable` as it doesn't really allow you to handle named arguments and scipy also uses a different parametrization.

Reply:
You can use a different parametrization in the tests, look at the [test for the gamma distribution for instance](https://github.com/aesara-devs/aesara/blob/c4330719564496f182cc7f8471c03512e76735f3/tests/link/jax/test_random.py#L124). Unless I misunderstood your problem?

Reply:
Nice, somehow missed that. I've pushed something to use `test_random_RandomVariable` but feels brittle to force the `loc` and `scale` arguments this way.